libft/sources/memory/ft_free_null.c

```diff
@@ -0,0 +1,34 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   ft_free_null.c                                     :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: taretiuk <taretiuk@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/10/23 12:34:16 by taretiuk          #+#    #+#             */
+/*   Updated: 2024/10/23 12:34:19 by taretiuk         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "../../libft.h"
+
+void	ft_free_ss_null(void ***c, int sz)
+{
+	if (c && *c)
```

Comment:
please revert the condition and add a return in it, so that current body becomes less indented